src/results/parse.jl

```diff
@@ -122,20 +159,12 @@ Adds power-based results.  See also [`get_table_summary`](@ref) for the below su
 | table_name | col_name | unit | description |
 | :-- | :-- | :-- | :-- |
 | :bus | :pgen | MWGenerated | Average Power Generated at this bus |
-| :bus | :egen | MWhGenerated | Electricity Generated at this bus for the weighted representative hour |
```

Comment:
why are we removing all of these?

Reply:
In short, to save memory and time.  I made several changes in this PR to make the serialized data.jls files smaller.  The egen, elserv, eflow, etc. are all easily computed from their power equivalent.  Choosing not to save `egen_gen` alone saves 12MB from each results file (assuming 30k generators, 52 repr hours).